Dockerfile

```diff
@@ -501,6 +501,96 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-libc as sdk-go-fips
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG SYSROOT="/${TARGET}/sys-root"
+ARG GOVER="1.21.4"
+ARG AWS_LC_FIPS_VER="2.0.2"
+
+USER root
+RUN DEBIAN_FRONTEND=noninteractive dnf install -y golang cmake xz wget ca-certificates clang python curl git
```

Comment:
We don't need `DEBIAN_FRONTEND=noninteractive`.

All of these packages should already be installed in the `sdk-libc` layer, which is based on `base` - except for `clang` and `golang`.

However, these programs in the list are somewhat suspicious:
* wget
* ca-certificates
* curl

Since they imply that we may be making outbound network requests. We have to pull all files and archives from the lookaside cache; we can't call out to upstream servers.

Can you make sure that there aren't any remote requests happening in either `/boring/build-boring.sh` or `/boring/build-goboring.sh`?

Reply:
I remove everything except for clang and golang and the build is working without problem. Don't think it's making output network requests during build.